### PR TITLE
Issue #3 - Unexpected end of JSON input

### PR DESCRIPTION
**Pull request by njparadis**
JSON.parse() throws an error if we just give an empty string. This change updates the response parsing to use an empty object if the API returns an empty body (which occurs on some delete methods).

### CODE REVIEW

**Review by remstone7 — APPROVED** [review]
probably will just want to add the changelog version bumo

**Review by smoucka — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### [Unreleased]
 
+### [1.0.2] 2021-01-12
```
Comment (smoucka) [inline CHANGELOG.md:9]: do we not do diff links for this changelog?

Comment (njparadis) [reply CHANGELOG.md:9]: I had just copied the style from the other SDK changelogs - we would need to start using tagged releases to support this I think. Do we know if there is a way to do that retroactively?

Comment (smoucka) [reply CHANGELOG.md:9]: Yea, you should just be able to tag previous commits. Definitely don't need to do this now.

Comment (njparadis) [reply CHANGELOG.md]: Beautiful - will tag this release properly as 1.0.2 so future diff links are less ugly.

**Review by smoucka — APPROVED** [review]

**Pull request merged**